### ISSUE TITLE
IGDM is not working

### ISSUE THREAD

**Issue by enes1993** [igdmapps/igdm#1426]
I opened the 3.0.2 v and 3 and 2.8 v ... But none of them opened. It was working before, but now it is not working. I tried old version 2.2 it was opened, but the passport screen was froze. So do you have any idea, what can I do?

https://igdm.me/?ref=producthunt (this is the app)

**Comment by Anzio1000** [comment]
agree, igmd is not working. Defective program with little or no support,

**Comment by ifedapoolarewaju** [comment]
what happens when you open it? What do you see on the screen?
